content/scripts/build.js

```diff
@@ -1236,11 +1231,16 @@ class Builder {
       return;
     }
 
+    // Get the summary from the raw HTML.
+    const rawHtml = fs.readFileSync(path.join(folder, "index.html"), "utf8");
+    const summary = getSummary(rawHtml);
```

Comment:
Perhaps because it's late but I can't comprehend this. 
That `rawHTML` is the *raw* HTML which is "like HTML" but it's not. It's full of KS macro snippets. Doesn't it seem very dangerous to hope this can be parsed by `cheerio`?
Also, will this make the `content/_all-titles.json` enormous?

One idea we should entertain is to move the summary out of the `index.html` and into the `index.yaml` but then I wonder, perhaps some summaries require KS macros expansion. 
Could it be an option to do something like this:
```javascript
let summary = metadata.summary
if (!summary) {
  logger.warning(`${folder}/index.yaml lacks a summary! Trying to extract from unrendered HTML`)
  doc.flaws.summary = "no summary"  // work-in-progress
}
```
That way we might consider circle back to the importer to extract this stuff. After all, the `wiki_document` has a`summary_html` and `summary_text` which I'm not super familiar with.

Reply:
We might want to pick Will's brain on this beause I know he's thought about it a lot in Stumptown.